docs/content/documentation/templates/overview.md

```diff
@@ -146,6 +146,21 @@ In the case of non-internal links, you can also add a cachebust of the format `?
 by passing `cachebust=true` to the `get_url` function.
 
 
+### 'get_file_hash`
```

Comment:
can that doc mention something about the search paths?

Reply:
Good point. I wrote a bit on this.